index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    />
+    <title>Thump a Trump</title>
+    <link
+      rel="stylesheet"
+      href="normalize.css"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="animations.css"
+      type="text/css"
+    />
+  </head>
+  <body>
+    <header>
+      <h1>Thump a Trump</h1>
+    </header>
+
+    <section class="menuScreen">
+      <img class="trumpy" src="images/trump01.svg" alt="" />
+
+      <a href="game.html">
+        <div class="startBtn">
+          <button>
```

Comment:
Use of the button tag is not allowed here, it's technically not allowed to be the child of an A tag according to the w3 spec. 

I'd also say that there isn't much point in using it as you're overriding all of the default button styles anyway!

Reply:
We're making it semantic and good for screen readers!

Reply:
You're going to hate me for it, but the w3 validator is now saying that the A tag is not allowed as a descendant of the button tag!

Buttons are only really meant for use inside of forms, you can use them as links but you need to do that using a JS event listener, which would then be ignored by a screen reader. They're mainly designed for form actions https://css-tricks.com/use-button-element/

Reply:
This arrangement also means that clicking the play icon doesn't work